chroma-manager/tests/integration/utils/chroma_log_collector.py

```diff
@@ -146,7 +146,8 @@ def fetch_iml_diagnostics(self, server):
             return["iml-diagnostics not installed on %s. skipping." % server]
 
         # Generate the diagnostics from the server
-        result = shell_run(['ssh', server, 'iml-diagnostics', '--all-logs'])
+        result = shell_run(
+            ['ssh', server, 'iml-diagnostics', '--all-logs'], timeout=300)
```

Comment:
I wonder if 5 minutes is enough.  I guess we can wait and see, but I would not be opposed to 10 minutes if you wanted.

I easy to wait and see if you prefer though.

Reply:
Sure, I'll up it.